ServerCore/Pages/Events/Standings.cshtml

```diff
@@ -10,6 +10,20 @@
 
 <h2>Standings</h2>
 
+@if(Model.Event.AllowsRemoteTeams)
+{ 
+    <p>
+        Filter standings to remote/local/all teams
+    </p>
+    <form method="GET" name="locationForm">
+        <select name="locationFilter" asp-items="@Model.TeamLocationFilter" onchange="this.form.submit();">
```

Comment:
Does submitting here preserve the sort order?

Reply:
It does not. Do you know of an easy way to do that with the dropdown form submit? I'm happy to make quick changes otherwise I'll file a bug for post-event.

Reply:
Nope, sorry. Does it *have* to be a drop-down? We have lots of examples of hyperlinks that add in all the needed properties, such as the sort links on many pages.